pkg/common/cns-lib/vsphere/hostsystem.go

```diff
@@ -95,15 +95,22 @@ type VsanHostCapacity struct {
 	HostMoID         string
 }
 
+type DiskHealth struct {
+	HealthFlags  int   `json:"healthFlags,omitempty"`
+	HealthReason int   `json:"healthReason,omitempty"`
+	TimeStamp    int64 `json:"timestamp,omitempty"`
```

Comment:
Seems like you are only using HealthFlags, can we not add other two?

Reply:
Actually I'm logging all of them below for debugging purpose.